src/ConnectedMode.UnitTests/Migration/MigrationPromptTests.cs

```diff
@@ -42,11 +41,10 @@ public class MigrationPromptTests
         public void MefCtor_CheckIsExported()
         {
             MefTestHelpers.CheckTypeCanBeImported<MigrationPrompt, IMigrationPrompt>(
-                MefTestHelpers.CreateExport<SVsServiceProvider>(),
```

Comment:
The class no longer uses `IServiceProvider` directly - it uses it indirectly via `ISolutionInfoProvider`.

As a consequence of this, it no longer needs to manage thread handling explicitly, since the implementation of `ISolutionInfoProvider` handles switching to the UI thread when necessary.